net-im/slack++/slack++-0.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="Slack Enterprise chat and services on the command line"
+DESCRIPTION="slack++ is an ncurses based client for the Searchable Log of All \
+Conversation and Knowledge."
+HOMEPAGE="https://github.com/ga2arch/slack"
+COPYRIGHT="2018 Federico Di Pierro"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitRev="f085f2a269d012c876e128db74d350754ec0ca30"
+SOURCE_URI="https://github.com/ga2arch/slack/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="2b42d26328e666dbbd431e8f389234eb19fd56f4ad65997d3d442b0cb96a4b6a"
+SOURCE_DIR="slack-$srcGitRev"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	slack++$secondaryArchSuffix = $portVersion
+	cmd:slack++$secondaryArchSuffix = $portVersion
```

Comment:
In case you choose to use the `commandBinDir` variable, the following suggestion overrides my previous one.
```diff
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
 SECONDARY_ARCHITECTURES="x86"
+commandBinDir=$prefix/bin
+fi
+
 PROVIDES="
 	slack++$secondaryArchSuffix = $portVersion
-	cmd:slack++$secondaryArchSuffix = $portVersion
+	cmd:slack++ = $portVersion
```

Reply:
Here's a variant for the first part of the patch in case you don't like the first part of the previous suggestion.
```diff
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
 SECONDARY_ARCHITECTURES="x86"
+fi
+
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandBinDir=$prefix/bin
+fi
```